### PR TITLE
Add more sample Instagram data files (CU-2vzcwe1)

### PR DESCRIPTION
**Pull request by Kahtaf**
### Description

- Adds 2 Instagram JSON files as sample data
- `JsonExtractor` modifies all object keys to lowercase, because some Instagram files have keys that are different casings. For example: 
```
"Contact syncing": {
   ...
},

vs

"Contact Syncing": {
   ...
},
```
Now, this key should be accessed by `obj["contact syncing"]`. 

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

File: src/utils/recursivelyLowercaseObjectKeys.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      prev[curr.toLowerCase()] = recursivelyLowercaseObjectKeys(
+        jsonDocument[curr]
+      );
+      return prev;
+    },
+    {}
+  );
+};
+
+const isArray = (obj: any) => {
```
Comment (RomanMeszarosToptal) [inline src/utils/recursivelyLowercaseObjectKeys.ts]: There is `Array.isArray()` in js, that receives as a parameter the object about which you want to know if it is an array. Do you want to keep this method? Does it take care of some special case?
When I changed it to `Array.isArray(obj)`, all tests were passing.

**Comment by RomanMeszarosToptal** [comment]
Looks good. I imported it to my branch, so it will be included in my next PR.
I had just one comment about the isArray function. Beside that it is a good idea to use the lower case here. (Another option that could work would be using a Proxy, but that would probably just add unnecessary complexity.)
I also changed the case on my branch where still needed.
I needed to do small changes in my code, as the additional zip files caused problems, that I quickly fixed.

**Review by RomanMeszarosToptal — APPROVED** [review]
This looks good, thanks.

File: src/utils/recursivelyLowercaseObjectKeys.ts (hunk 1 above)
Comment (Kahtaf) [reply src/utils/recursivelyLowercaseObjectKeys.ts]: Nice catch, I changed it.

**Pull request merged**